src/main/java/org/gridsuite/explore/server/StudyService.java

```diff
@@ -50,10 +50,10 @@ public void setStudyServerBaseUri(String studyServerBaseUri) {
         this.studyServerBaseUri = studyServerBaseUri;
     }
 
-    public Mono<Void> insertStudyWithExistingCaseFile(UUID studyUuid, String studyName, String description, String userId, Boolean isPrivate, UUID caseUuid) {
+    public Mono<Void> insertStudyWithExistingCaseFile(UUID studyUuid, String userId, Boolean isPrivate, UUID caseUuid) {
         String path = UriComponentsBuilder.fromPath(DELIMITER + STUDY_SERVER_API_VERSION +
-                        "/studies/{studyName}/cases/{caseUuid}?description={description}&isPrivate={isPrivate}&studyUuid={studyUuid}")
-                .buildAndExpand(studyName, caseUuid, description, isPrivate, studyUuid)
+                        "/studies/cases/{caseUuid}?isPrivate={isPrivate}&studyUuid={studyUuid}")
```

Comment:
Maybe it would be better to use a URL like this:
"/studies/{studyUuid}/cases/{caseUuid}?isPrivate={isPrivate}"
or do we want to allow creating a study without specifying its studyUuid ?

Reply:
After  thinking about it, we might want to create a study without specifying its uuid. I close this conversation.